automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala

```diff
@@ -161,7 +161,7 @@ final class NotebookCustomizationSpec extends GPAllocFixtureSpec with ParallelTe
             GcsRoles.Owner)
 
           withNewCluster(billingProject,
-            request = defaultClusterRequest.copy(jupyterStartUserScriptUri = Some(startScriptUri))) {
+            request = defaultClusterRequest.copy(jupyterStartUserScriptUri = Some(startScriptUri), jupyterUserScriptUri = Some(startScriptUri))) {
```

Comment:
Why add this? Now I expect this script to be run 3x, so the test should hopefully fail